### ISSUE TITLE
Combobox with False-positive/Intended/... is not shown on last revision

### ISSUE THREAD

**Issue by irishrover** [Ericsson/codechecker#1863]
**Describe the bug**
The combobox for False positive/Intended/... is not shown in Chrome/Chromium, only in Firefox.

**CodeChecker version**
I'm on 
commit 888903da5868f684c1e3b7bebbac3230a5880b18 (HEAD -> master, origin/master, origin/HEAD)
Merge: 5363e384 fd4aa726
Author: Gyorgy Orban <o.gyorgy@gmail.com>
Date:   Mon Jan 21 09:15:59 2019 +0100
    Merge pull request #1818 from csordasmarton/show-macro-and-notes
    Show macro expansions and notes

**To Reproduce**
1. Just open any file report with issues with Chrome/Chromium, not Firefox. You'll notice there's no combobox between `Show documentation` button and `Show arrows`.

**Desktop (please complete the following information)**
 - OS: Windows 10, Ubuntu 16.
 - Browser Chrome/Chromium
 - Version Chrome 71.0.3578.98 (Official Build) (64-bit) (cohort: Stable)

**Additional context**
It worked few days ago.

**Comment by csordasmarton** [comment]
Hi @irishrover,

There was a bug in a migration script. I fixed it in #1869. To solve your problem please update your git repository to the latest and run the following SQL query on your product configuration database:
```sql
UPDATE products SET is_review_status_change_disabled=0
```

**Comment by irishrover** [comment]
It helped! Thanks!
